x-pack/plugins/ml/public/formatters/metric_change_description.js

```diff
@@ -67,19 +72,30 @@ export function getMetricChangeDescription(actualProp, typicalProp) {
         if (factor <= 100) {
           message = `${Math.round(factor)}x ${direction}`;
         } else {
-          message = `More than 100x ${direction}`;
+          message = i18n.translate('xpack.ml.formatters.metricChangeDescription.moreThan100xDescription', {
+            defaultMessage: 'More than 100x {direction}',
```

Comment:
It doesn't look like `direction` is getting translated. `higher` or `lower` should be translated too - a few lines above

Reply:
translated them. @peteharverson could you please check one more time?